File: Array/03_check_array_sorted/sorted_array.py
def sorted_array(arr):
    for i in range(len(arr)-1):
        if arr[i] > arr[i+1]: 
            return False
    return True

print(sorted_array([1, 2, 3, 4])) 
print(sorted_array([1, 3, 2, 4]))

File: Array/03_check_array_sorted/sorted_array.js
function sorted_array(arr) {
  for (let i = 1; i < arr.length; i++) {
    if (arr[i] >= arr[i - 1]) {
    } else {
      return false;
    }
  }
  return true;
}

arr = [1, 2, 3, 4, 5, 6, 7];

console.log(sorted_array(arr));
